api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 265**
```

Comment:
We have been using colon after this line in some of the recently added feature levels 
like `**Feature level 263**:`. Not sure if that's intended and we should it do it for all the future feature levels.